platform/ios/src/MGLScaleBar.mm

```diff
@@ -0,0 +1,384 @@
+#import <Mapbox/Mapbox.h>
+#import "MGLScaleBar.h"
+
+static const CGFloat MGLFeetPerMile = 5280;
+
+struct MGLRow {
+    CLLocationDistance distance;
+    NSUInteger numberOfBars;
+};
+
+static const struct {
+    struct MGLRow row;
```

Comment:
Anyhow, since the Imperial table needs access to the same type as this table, it does make sense to name MGLRow, just not to then stuff it into an anonymous struct as this code does.